### PR TITLE
Added Components

### PR DESCRIPTION
**Pull request by b0llu**
## Files to check
- [ ] src/Components/Footer/Footer.jsx
- [ ] src/Components/Header/Header.jsx
- [ ] src/Components/LandingContainer/LandingContainer.jsx
- [ ] src/Components/ProductContainer/ProductContainer.jsx
- [ ] src/Components/Sidebar/BrandFilter/BrandFilter.jsx
- [ ] src/Components/Header/Header.jsx
- [ ] src/Components/Sidebar/CategoryFilter/CategoryFilter.jsx
- [ ] src/Components/Sidebar/FilterHeader/FilterHeader.jsx
- [ ] src/Components/Sidebar/OtherFilter/OtherFilter.jsx
- [ ] src/Components/Sidebar/RatingFilter/RatingFilter.jsx
- [ ] src/Components/Sidebar/Sidebar.jsx
- [ ] src/Components/Sidebar/SortFilter/SortFilter.jsx
- [ ] src/Components/Toast/Toast.jsx

## What not to check
- [ ] Do not check the useEcom hook or the dispatch action in toast
- [ ] Do not check App.js or Index.js
- [ ] Do not check yarn.lock

Check the whole site below,
https://wizardly-dijkstra-e641bc.netlify.app/

### CODE REVIEW

File: src/Components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+              <div className="number">{forLength.wishlistLength}</div>
+            </div>
+            <div className="badge">
+              <Link to="/Cart" element={<CartPage />}>
+                <i className="fas fa-shopping-cart"></i>
+              </Link>
+              <div className="number">{forLength.cartLength}</div>
+            </div>
+            <div className="badge">
+              <a>
```
Comment (hsnice16) [inline src/Components/Header/Header.jsx:38]: As this `<a>` has no `href` so you can remove this, and in place of this, you can use `span`.

**Review by hsnice16 — APPROVED** [review]
Everything looks good. Nice work. Just added a few suggestions.

File: src/Components/Header/Header.jsx (hunk 1 above)
Comment (b0llu) [reply src/Components/Header/Header.jsx:38]: thanks for pointing it out will change it

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,25 @@
...
 function App() {
   const [someData, setSomeData] = useState([]);
 
   useEffect(() => {
     (async function () {
-      const { data } = await axios.get("/api/brands");
-      setSomeData(data.brands);
+      const { data } = await axios.get("/api/products");
+      setSomeData(data.products);
       console.log(data);
```
Comment (abhisheksahani-nova) [inline src/App.js:13]: Remove console.log after being used

File: src/App.js [hunk 3]
```diff
@@ -1,24 +1,25 @@
 import axios from "axios";
 import { useState } from "react";
 import { useEffect } from "react";
-import "./App.css";
+import { Footer, Sidebar } from "./Components";
 
 function App() {
   const [someData, setSomeData] = useState([]);
```
Comment (abhisheksahani-nova) [inline src/App.js:7]: It will better if you give better naming for your states , eg- products

File: src/Components/Sidebar/Sidebar.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      <div
+        id="sidebar-menu"
+        className="sidebar stick position-open"
+        // className={`sidebar stick ${
+        //   sidebar ? "position-open" : "position-hidden"
+        // }`}
+      >
+        <FilterHeader />
+        <ul className="list-style-none filter-section">
+          <div className="filter-divider-line"></div>
```
Comment (abhisheksahani-nova) [inline src/Components/Sidebar/Sidebar.jsx:24]: For divider line , you can use <hr />, or border-bottom property on existing div

**Review by abhisheksahani-nova — APPROVED** [review]
Code looks good, just look at the suggested changes once to make necessary changes , Dhruv great work  though!

File: src/App.js (hunk 2 above)
Comment (b0llu) [reply src/App.js:13]: yes this app.js is just for testing purposes it will be cleared later

File: src/App.js (hunk 3 above)
Comment (b0llu) [reply src/App.js:7]: its just for testing purposes

File: src/Components/Sidebar/Sidebar.jsx (hunk 4 above)
Comment (b0llu) [reply src/Components/Sidebar/Sidebar.jsx:24]: thanks for the suggestion

**Pull request merged**